src/main/java/me/totalfreedom/totalfreedommod/command/Command_clearmychat.java

```diff
@@ -0,0 +1,23 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = Rank.OP, source = SourceType.BOTH)
+@CommandParameters(description = "This is a for the OP's that want to clear there chat", usage = "/<command>", aliases = "cmc")
+public class Command_clearmychat extends FreedomCommand
+{
+    @Override
+    protected boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        for (int i = 0; i <= 150; i++)
+        {
+            msg("");
```

Comment:
This will make server lag because of message spam, nice idea savinith! There alot of ways to crash a server with a command, this will force console to spam that message, you should use spigot methods to do client-side message without harming server